test/extended/crc/ux/notification/notification_windows.go

```diff
@@ -15,11 +15,11 @@ type gowinxHandler struct {
 }
 
 const (
-	startMessage  string = "CodeReady Containers Cluster has started"
+	startMessage  string = "CRC Cluster has started"
```

Comment:
This seems to be coming from `tray-windows`, not `tray-electron`, do we still need to test this?

Reply:
tray notifications show "Red Hat OpenShift Local" if this is still tested.

Reply:
Thanks everyone, this is then probably true for all 4 files in here, including the macos installer window I assume? I'll see if we still run these (but CodeReady Containers should probably not appear in our code whether we run it or not...)

Reply:
@gbraad @cfergeau @anjannath I changed them all to _OpenShift Local_. Not sure if we'll be using this code, but changing anyways to not have "CodeReady Containers" in the repo.